sailfishusermanager.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020 Jolla Ltd.
+ * Copyright (c) 2020 Open Mobile Platform LLC.
+ *
+ * All rights reserved.
+ *
+ * BSD 3-Clause License, see LICENSE.
+ */
+
+#ifndef SAILFISHLOGIND_H
+#define SAILFISHLOGIND_H
+
+#include <QDBusContext>
+#include <QDBusArgument>
+
+class QTimer;
+class LibUserHelper;
+
+struct SailfishUser {
```

Comment:
These things needed to use the D-Bus interface should probably be in some separate header file and available in devel subpackage. This struct, those operator methods below and definitions for D-Bus service names, interfaces and paths at least.

Reply:
Devel package added.